libs/components/src/popover/vl-floating-ui.controller.ts

```diff
@@ -69,30 +69,32 @@ export default class FloatingController implements ReactiveController {
     }
 
     async updatePosition(): Promise<void> {
-        const { x, y, strategy, placement, middlewareData } = await computePosition(
-            this.getReferenceElement()!,
-            this.host,
-            {
-                placement: this.options.placement,
-                middleware: this.buildMiddlewares(),
-            }
-        );
-
-        Object.assign(this.host.style, {
-            position: strategy,
-            left: `${x}px`,
-            top: `${y}px`,
-        });
-
-        const staticSide = { top: 'bottom', right: 'left', bottom: 'top', left: 'right' }[placement.split('-')[0]];
-
-        if (middlewareData.arrow && this.getArrowElement()) {
-            const { x, y } = middlewareData.arrow;
-            Object.assign(this.getArrowElement().style, {
-                left: x != null ? `${x}px` : '',
-                top: y != null ? `${y}px` : '',
-                [staticSide!]: `${-this.getArrowElement().offsetWidth / 2}px`,
+        if (this.getReferenceElement) {
```

Comment:
ik zou hier fast failen.
M.a.w: returnend indien niet gezet
```suggestion
        if (!this.getReferenceElement) {
          return
        }
```